src/main/java/uk/gov/pay/webhooks/app/WebhookMessageDeletionConfig.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.pay.webhooks.app;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+public class WebhookMessageDeletionConfig {
+    
+    @Valid
+    @NotNull
+    private int maxAgeOfMessages;
```

Comment:
looks like the `config.yaml` entry for these vars is missing

Reply:
It's not really needed just yet as the default values are 0 and there's no endpoint to trigger the `WebhookService.deleteWebhookMessagesOlderThan` just yet. The important thing is that the integration tests pass.

Reply:
no worries, just weren't sure if you were implementing the second part of this story!